docs/index.rst

```diff
@@ -84,6 +86,16 @@ sphere.
 License
 *******
 
-While ``pyvistaqt`` is under the MIT license, ``pyqt5`` is subject to
-the GPL license.  Please see deails at
+``pyvistaqt`` is under the MIT license.
+However, Qt bindings have licenses of their own.
+
+Historically, ``pyvistaqt`` has used ``pyqt5``, which is subject
+to the GPL license. See details at
 `Riverbank License FAQ <https://www.riverbankcomputing.com/commercial/license-faq>`_.
+
+``pyvistaqt`` is transitioning to using ``qtpy``
+
+> QtPy is a small abstraction layer that lets you write applications using a single API call to either PyQt or PySide.
```

Comment:
Why the leading `>`? I would make this more compact like:
```suggestion
Historically, ``pyvistaqt`` has used ``pyqt5``, which is subject
to the GPL license. See details at
`Riverbank License FAQ <https://www.riverbankcomputing.com/commercial/license-faq>`_. ``pyvistaqt`` now uses QtPy if present, and will require it after version 0.3. QtPy is a small abstraction layer that lets you write applications using a single API call to either PyQt or PySide (which has a LGPL license).
```
The wording about being required or not should be adjusted based on whether or not there is actually going to be a deprecation period.

Reply:
The leading > is because it's a quote from the qtpy docs... But I'm open to changing it to something that looks nicer.